keeper/src/main/scala/zio/keeper/membership/swim/SWIM.scala

```diff
@@ -24,7 +25,7 @@ import scala.collection.immutable.SortedSet
 
 object SWIM {
 
-  def join(
+  def live(
```

Comment:
I would even add aliases to these environments, as they are getting a bit out of hand.